### PR TITLE
modulesync 5.2.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 5.0.0

### CODE REVIEW

File: spec/defines/resource_upstream_spec.rb [hunk 1]
```diff
@@ -36,6 +38,7 @@
         ]
       end
 
+      # rubocop:disable RSpec/MultipleMemoizedHelpers
```
Comment (root-expert) [inline spec/defines/resource_upstream_spec.rb:41]: Maybe that will make sense to disable it globally?

Comment (smortex) [reply spec/defines/resource_upstream_spec.rb:41]: I was thinking about it but I have mixed feelings: on one side it is a smell that really should be avoided, but at the same time in acceptance tests it is something ugly for which I can't find a clean solution.

Disabling these checks here is the easy way to move on quickly without blocking everything, not something I expect to see generalized.

File: spec/classes/nginx_spec.rb [hunk 2]
```diff
@@ -36,26 +38,29 @@
         it { is_expected.to contain_nginx__resource__streamhost('streamhost1').with_proxy('streamproxy') }
       end
 
-      context 'nginx::package' do
+      context 'nginx::package' do # rubocop:disable RSpec/EmptyExampleGroup
```
Comment (ekohl) [inline spec/classes/nginx_spec.rb:41]: Once https://github.com/rubocop/rubocop-rspec/pull/1249 is released, this should no longer be needed

File: spec/acceptance/nginx_mail_spec.rb [hunk 3]
```diff
@@ -78,9 +83,10 @@ class { 'nginx':
 
       apply_manifest(pp, catch_failures: true)
     end
+
     describe file('/etc/nginx/conf.mail.d/domain1.example.conf') do
       it 'does\'t contain `ssl` on `listen` line' do
-        is_expected.to contain 'listen                *:465;'
+        expect(subject).to contain 'listen                *:465;'
```
[the remark this answers is not in the document]
Comment (bastelfreak) [reply spec/acceptance/nginx_mail_spec.rb:89]: we applied this change before we updated voxpupuli-test. I'm currently trying to revert it locally.

**Pull request closed without merging**